NOMIS/nomis.go

```diff
@@ -109,13 +116,32 @@ func main() {
 		return
 	}
 
-	for index0, _ := range res.Structure.Keyfamilies.Keyfamily {
+	for index0 := range res.Structure.Keyfamilies.Keyfamily {
```

Comment:
Not sure index0 makes much sense here, it is going to change after every iteration (completing the `range`) index0 as a variable implies to me It has the value `0` at all times and is used to get some item at index0 - but I don't think that is what it really is doing here but rather represents a Key, in this case the Keyfamily